src/main/java/org/openmbee/mdk/emf/EMFBulkImporter.java

```diff
@@ -333,4 +340,62 @@ public Map<Pair<Element, ObjectNode>, Exception> getFailedElementMap() {
     public Map<Element, ObjectNode> getNonEquivalentElements() {
         return nonEquivalentElements;
     }
+
+    // //Shim for 2022x R2 until we can figure out if it's still ncessary
+    // public static boolean isElementInvalid(Element element) {
+    //     //For 2021x and earlier
+    //     //return element.isInvalid();
+    //     return false;
+    // }
+    
+    // Is Element Invalid Shim for versions earlier that 2021x
+    private static boolean isElementInvalid(Element element) {
```

Comment:
sounds like this has always just returned false, can just take this out everywhere